{"uid":"1d7d997d360127b1","name":"Overflow of the sum of two fractions","fullName":"FractionTest.longFraction","historyId":"5c34ac3893aa9f16f89df3e4004083a7","time":{"start":1747803292376,"stop":1747803292384,"duration":8},"description":"","status":"failed","statusMessage":"An exception should be thrown. ArithmeticException ==> Unexpected exception type thrown, expected: <java.lang.ArithmeticException> but was: <java.lang.RuntimeException>","statusTrace":"org.opentest4j.AssertionFailedError: An exception should be thrown. ArithmeticException ==> Unexpected exception type thrown, expected: <java.lang.ArithmeticException> but was: <java.lang.RuntimeException>\n\tat org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)\n\tat org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)\n\tat org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:39)\n\tat org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3131)\n\tat FractionTest.longFraction(FractionTest.java:54)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)\n\tat java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)\n\tat java.base/java.util.concurrent.ForkJoinTask.awaitDone(ForkJoinTask.java:436)\n\tat java.base/java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:670)\n\tat java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)\n\tat java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)\n\tat java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)\n\tat java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)\n\tat java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)\n\tat java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)\nCaused by: java.lang.RuntimeException: Common denominator is too big!\n\tat uz.kamoliddin.org.fpmi.Fraction.sum(Fraction.java:31)\n\tat FractionTest.lambda$longFraction$1(FractionTest.java:54)\n\tat org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:53)\n\t... 14 more\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"tag","value":"Negative"},{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:FractionTest]/[method:longFraction()]"},{"name":"host","value":"fv-az788-957"},{"name":"thread","value":"2647@fv-az788-957.ForkJoinPool-1-worker-4(20)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"FractionTest"},{"name":"testClass","value":"FractionTest"},{"name":"testMethod","value":"longFraction"},{"name":"suite","value":"FractionTest"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":6,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":6},"items":[{"uid":"39248693c90648a2","reportUrl":"https://ravshanovkamoliddin.github.io/unitTestsGradle/14//#testresult/39248693c90648a2","status":"failed","statusDetails":"An exception should be thrown. ArithmeticException ==> Unexpected exception type thrown, expected: <java.lang.ArithmeticException> but was: <java.lang.RuntimeException>","time":{"start":1747801481065,"stop":1747801481071,"duration":6}},{"uid":"4cf2f355fd1debf","reportUrl":"https://ravshanovkamoliddin.github.io/unitTestsGradle/13//#testresult/4cf2f355fd1debf","status":"failed","statusDetails":"An exception should be thrown. ArithmeticException ==> Unexpected exception type thrown, expected: <java.lang.ArithmeticException> but was: <java.lang.RuntimeException>","time":{"start":1747801393038,"stop":1747801393054,"duration":16}},{"uid":"b88c9153c976d5db","reportUrl":"https://ravshanovkamoliddin.github.io/unitTestsGradle/12//#testresult/b88c9153c976d5db","status":"failed","statusDetails":"An exception should be thrown. ArithmeticException ==> Unexpected exception type thrown, expected: <java.lang.ArithmeticException> but was: <java.lang.RuntimeException>","time":{"start":1747066176731,"stop":1747066176734,"duration":3}},{"uid":"47cf580d72aa9350","reportUrl":"https://ravshanovkamoliddin.github.io/unitTestsGradle/11//#testresult/47cf580d72aa9350","status":"failed","statusDetails":"An exception should be thrown. ArithmeticException ==> Unexpected exception type thrown, expected: <java.lang.ArithmeticException> but was: <java.lang.RuntimeException>","time":{"start":1746944925857,"stop":1746944925873,"duration":16}},{"uid":"55e7c2f436a82b95","reportUrl":"https://ravshanovkamoliddin.github.io/unitTestsGradle/10//#testresult/55e7c2f436a82b95","status":"failed","statusDetails":"An exception should be thrown. ArithmeticException ==> Unexpected exception type thrown, expected: <java.lang.ArithmeticException> but was: <java.lang.RuntimeException>","time":{"start":1746944914319,"stop":1746944914337,"duration":18}}]},"tags":["Negative"]},"source":"1d7d997d360127b1.json","parameterValues":[]}